<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="image_display">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="debug_image_display">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="contour_image_display">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>60</y>
      <width>1200</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
